chef_master/source/resource_windows_service.rst

```diff
@@ -287,3 +332,111 @@ The following examples demonstrate various approaches for using resources in rec
    end
 
 .. end_tag
+
+**Create a service**
+
+.. tag resource_service_windows_create
+
+.. To create service with 'name':
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :create
+     binary_path_name "C:\\opscode\\chef\\bin"
+   end
+
+Create service with 'service_name' and 'display_name':
+
+.. code-block:: ruby
+
+   windows_service 'Create chef client as service' do
+     action :create
+     display_name "CHEF-CLIENT"
+     service_name "chef-client"
+     binary_path_name "C:\\opscode\\chef\\bin"
+   end
+
+Create service with the ``:manual`` startup type:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :create
+     binary_path_name "C:\\opscode\\chef\\bin"
+     startup_type :manual
+   end
+
+Create a service with the ``:disabled`` startup type:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :create
+     binary_path_name "C:\\opscode\\chef\\bin"
+     startup_type :disabled
+   end
+
+Create service with the ``:automatic`` startup type and delayed start enabled:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :create
+     binary_path_name "C:\\opscode\\chef\\bin"
+     startup_type :automatic
+     delayed_start true
+   end
+
+Create service with a description:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :create
+     binary_path_name "C:\\opscode\\chef\\bin"
+     startup_type :automatic
+     description "Chef client as service"
+   end
+
+.. end_tag
+
+**Delete a service**
+
+.. tag resource_service_windows_delete
+
+Deelete service with the ``'name'`` of ``chef-client``:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :delete
+   end
+
+Delete service with ``'service_name'``:
+
+.. code-block:: ruby
+
+   windows_service 'Delete chef client' do
+     action :delete
+     service_name "chef-client"
+   end
+
+.. end_tag
+
+**Configure a service**
+
+.. tag resource_service_windows_configure
+
+Change an existing service from automatic to manual startup:
+
+.. code-block:: ruby
+
+   windows_service 'chef-client' do
+     action :configure
+     binary_path_name "C:\\opscode\\chef\\bin"
+     startup_type :manual
+     delayed_start true
```

Comment:
If I'm reading this correctly, `delayed_start` isn't going to work if `startup_type` is set to `:manual`.